_source/_layouts/author.html

```diff
@@ -35,6 +35,8 @@ <h1 class="BlogPost-title">
 		    </div>
 		{% endif %}
 
+    <h4>All Posts by {{ page.full_name }}</h4>
```

Comment:
The only header above this is an h1. Not sure it's a good idea to go from h1 to h4. h2 might be better?

Reply:
I *think* it's better this way, because it's meant to be unimportant to the SEOz. AKA: it isn't like some category/section header I want optimized. It's just supposed to be a small, unimportant heading that is a visual indicator to the reader. At least that's what I was going for.

If we do h2 it'll be huge and set off all sorts of SEO flags for the search engines =/ I could alternatively do a paragraph with custom styling, but that felt sorta hacky.